GVFS/GVFS/CommandLine/ConfigVerb.cs

```diff
@@ -31,30 +48,70 @@ protected override string VerbName
 
         public override void Execute()
         {
-            this.localConfig = new LocalGVFSConfig();
+            if (GVFSPlatform.Instance.IsUnderConstruction)

```

Comment:
Can you make this same change in `UpgradeVerb`?

Reply:
Sure... but this is not related to gvfs config. This task is there in the clean-up TODO list. Will have that PR up soon.